recipe/0000-setup.py-version.patch

```diff
@@ -0,0 +1,12 @@
+diff -ruN esmf-ESMF_8_1_1.orig/src/addon/ESMPy/setup.py esmf-ESMF_8_1_1/src/addon/ESMPy/setup.py
+--- esmf-ESMF_8_1_1.orig/src/addon/ESMPy/setup.py	2021-04-23 00:21:45.000000000 +0200
++++ esmf-ESMF_8_1_1/src/addon/ESMPy/setup.py	2021-09-21 17:52:19.798760790 +0200
+@@ -242,7 +242,7 @@
+ # TODO: build doc command
+ # TODO: remove duplicated metadata: here and src/ESMF/__init__.py
+ setup(name="ESMPy",
+-      version="8.1.0",
++      version="8.1.1",
```

Comment:
I was wondering about that. Upstream is already at 8.2.0 (in main), set correctly in the setup.py. But of course it might be worth keeping track, will do.

Reply:
They don't allow the opening of issues, but I informed them by email.